jest.config.json

```diff
@@ -2,12 +2,20 @@
   "clearMocks": true,
   "moduleDirectories": ["node_modules", "src"],
   "moduleFileExtensions": ["js", "jsx"],
+  "moduleNameMapper": {
+    "^.+\\.(gif|jpg|jpeg|png)$": "<rootDir>/mocks/fileMock.js",
```

Comment:
fileMock.js is basically just an empty file to allow for images to be important, but basically ignored.

Reply:
Yes, sorry :D  Comment fixed.